packages/features/bookings/Booker/components/DatePicker.tsx

```diff
@@ -35,7 +36,7 @@ export const DatePicker = ({
         setMonth(date.format("YYYY-MM"));
         setSelectedDate(date.format("YYYY-MM-DD"));
       }}
-      includedDates={nonEmptyScheduleDays}
+      includedDates={BOOKER_NUMBER_OF_DAYS_TO_LOAD > 0 ? null : nonEmptyScheduleDays}
```

Comment:
This makes it such the entire calendar is clickable and data is loaded in once clicked.